local.conf.example_staging

```diff
@@ -0,0 +1,154 @@
+ADMIN_PASSWORD=nomoresecret
+DATABASE_PASSWORD=123
+#RABBIT_USERID=clearstack
+RABBIT_PASSWORD=123
+SERVICE_PASSWORD=123
+
+USE_PYTHON3=False
+ENABLED_PYTHON3_PACKAGES=""
+
+LIBS_FROM_GIT=python-cinderclient,python-glanceclient,python-neutronclient,python-novaclient,python-openstacksdk,neutron-lib
+
+STX_BASE=https://github.com/starlingx-staging
+
+# major services
+CINDER_REPO=${STX_BASE}/stx-cinder.git
+CINDER_BRANCH=master
+GLANCE_REPO=${STX_BASE}/stx-glance.git
+GLANCE_BRANCH=master
+HORIZON_REPO=${STX_BASE}/stx-horizon.git
+HORIZON_BRANCH=master
+KEYSTONE_REPO=${STX_BASE}/stx-keystone.git
+KEYSTONE_BRANCH=master
+NEUTRON_REPO=${STX_BASE}/stx-neutron.git
+NEUTRON_BRANCH=master
+NOVA_REPO=${STX_BASE}/stx-nova.git
+NOVA_BRANCH=master
+# python clients
+CINDERCLIENT_REPO=${STX_BASE}/stx-python-cinderclient.git
+CINDERCLIENT_BRANCH=master
+GLANCECLIENT_REPO=${STX_BASE}/stx-python-glanceclient.git
+GLANCECLIENT_BRANCH=master
+NEUTRONCLIENT_REPO=${STX_BASE}/stx-python-neutronclient.git
+NEUTRONCLIENT_BRANCH=master
+NOVACLIENT_REPO=${STX_BASE}/stx-python-novaclient.git
+NOVACLIENT_BRANCH=master
+OPENSTACKSDK_REPO=${STX_BASE}/stx-openstacksdk.git
+OPENSTACKSDK_BRANCH=master
+NEUTRON_LIB_REPO=${STX_BASE}/stx-neutron-lib.git
+NEUTRON_LIB_BRANCH=master
+
+
+
+CELLSV2_SETUP=singleconductor
+# Override the ENABLED_SERVICES
+ENABLED_SERVICES=key,postgresql,n-api,n-cpu,n-cond,n-sch,n-cauth,n-api-meta
+enable_service rabbit
+enable_service placement-api,placement-client
+enable_service g-api,g-reg
+enable_service q-svc,q-dhcp,q-meta,q-agt,q-l3,q-trunk
+enable_service c-api,c-sch
+#enable_service c-api,c-sch,c-vol
+#enable_service horizon
+# ``HOST_IP`` and ``HOST_IPV6`` should be set manually for best results if
+# the NIC configuration of the host is unusual, i.e. ``eth1`` has the default
+# route but ``eth0`` is the public interface.  They are auto-detected in
+# ``stack.sh`` but often is indeterminate on later runs due to the IP moving
+# from an Ethernet interface to a bridge on the host. Setting it here also
+# makes it available for ``openrc`` to include when setting ``OS_AUTH_URL``.
+# Neither is set by default.
+#HOST_IP=w.x.y.z
+#HOST_IPV6=2001:db8::7
+
+enable_plugin networking-sfc https://github.com/starlingx-staging/stx-networking-sfc.git
+
+# Logging
+# -------
+
+# By default ``stack.sh`` output only goes to the terminal where it runs.  It can
+# be configured to additionally log to a file by setting ``LOGFILE`` to the full
+# path of the destination log file.  A timestamp will be appended to the given name.
+LOGDIR=/opt/stack/logs
+LOGFILE=/opt/stack/logs/stack.sh.log
+
+# Old log files are automatically removed after 7 days to keep things neat.  Change
+# the number of days by setting ``LOGDAYS``.
+LOGDAYS=2
+
+# Nova logs will be colorized if ``SYSLOG`` is not set; turn this off by setting
+# ``LOG_COLOR`` false.
+#LOG_COLOR=False
+
+# Using milestone-proposed branches
```

Comment:
I would remove this section, we don't have anything like milestone-proposed and it duplicates the branch setting s from above